### ISSUE TITLE
Unable to install in XX version of chromium

### ISSUE THREAD

**Issue by NeverDecaf** [NeverDecaf/chromium-web-store#10]
There have been several reports of being unable to install this extension in a certain version or derivative of Chromium. This is most likely an issue with your particular browser and outside the scope of this extension. If you are able to install other **non-webstore** extensions but not this one **in particular** then please open a separate issue, otherwise try following the steps below or commenting on this issue to see if other users can help.
1. Download the .crx from releases and extract the contents to a folder
2. Visit chrome://extensions/ and turn on developer mode (toggle in top right)
3. Click `Load unpacked` and select the directory you extracted the crx to.

**Comment by woolyss** [comment]
Hello @NeverDecaf ! ;)

Hope all is good for you! ;)

Today I made a video to prove that your extension works nicely in Chromium Ungoogled v80, on Windows 10. 😉👍

System:
- Chromium Ungoogled 80.0.3987.132 (64-bit version from https://chromium.woolyss.com/)
- Windows 10 OS Version 1909 (Build 18363.657)
- Installation of `Ublock origin` via your [chromium-web-store](https://github.com/NeverDecaf/chromium-web-store) extension.

You are free to use the following links! ;)

- My own storage: https://chromium.woolyss.com/f/video-chromium-web-store-for-chromium-ungoogled-v80.mp4 (Size: 4 MB)
- Mirror: https://streamable.com/4mhg6

**Comment by jstkdng** [comment]
@woolyss Does Marmaduke release the source of the tarball they use to build?

**Comment by woolyss** [comment]
@jstkdng: Yes. You can find the source code at https://github.com/Eloston/ungoogled-chromium/

The only thing different and not shared by Marmaduke himself is the support for the h.265 codec. Don't worry, it's not used by any website. Is is normally used on Chromium OS.

**Comment by SurlyVagrant** [comment]
I just set up Marmaduke's ungoogled Chromium 83.0.4103.97 (756066) and was having problems getting chromium-web-store working, but I think I figured out the process to make it work.

Problem 1:
Drag and dropping the chromium-web-store crx into the extensions window failed with the message "Package is invalid: 'CRX_REQUIRED_PROOF_MISSING'".
Solution: Turn on developer mode and refresh the extensions page if it is open. It should now let you drag and drop the crx into the extension window. No need to extract the crx and you can disable it again once chromium-web-store is installed.

Problem 2:
Trying to install another extension from the store would say the website wasn't allowed to install extensions. 
Solution: In the ungoogled FAQ under the [Installing the CRX](https://ungoogled-software.github.io/ungoogled-chromium-wiki/faq#installing-the-crx-file) heading it says "Change the flag chrome://flags/#extension-mime-request-handling to Always prompt for install." Once the flag is changed and chromium restarts, it should properly prompt to install extensions.

In theory you could use the solution to problem 1, then use one of the other methods in the ungoogled FAQ to download and install extensions, but this seems to work the way you would expect it.

**Comment by woolyss** [comment]
@SurlyVagrant:
I tested with the latest version of   [chromium-web-store extension](https://github.com/NeverDecaf/chromium-web-store).

The chromium-web-store extension works nicely to install extensions from the [Chrome Web Store](https://chrome.google.com/webstore/)
 but **the drag-and-drop currently doesn't work** in Ungoogled Chromium (83.0.4103.97 by marmaduke). This is an issue of  Ungoogled Chromium.

1. chrome://flags/#extension-mime-request-handling --> Always prompt for install
 --> Then, restart the browser

2. chrome://extensions --> Click on "Developer mode" to enable it.

3. Download the **chromium-web-store** extension from its [releases](https://github.com/NeverDecaf/chromium-web-store/releases)

4. Extract the CRX content where you want. 

5. chrome://extensions --> Click on "Load Unpacked" to load the extension (CRX content).

6. Go to the [Chrome Web Store](https://chrome.google.com/webstore/) and install all extensions needed. Note: When you click on the "Add to Chromium" button, a prompt window will display before to install the extension.

**Comment by SurlyVagrant** [comment]
Drag and drop does seem to work, it's just a little finicky. My copy of chromium that is already set up with all my extensions and stuff let's me drag and drop even with developer mode off for some reason, so I downloaded a fresh copy of the portable release to test.

1. Drag and drop the crx into the extensions window on the fresh copy --> gives the CRX_REQUIRED_PROOF_MISSING error.

2. Turn on Developer Mode.

3. Drag and drop the crx without refreshing the window --> still gives the CRX_REQUIRED_PROOF_MISSING error.

3. With Developer Mode still on, press F5 to refresh the extensions window.

4. Drag and drop the crx into the extensions window and it installs correctly.

So something about developer mode will let you drag and drop, you just have to remember to refresh the window after you turn developer mode on. If that doesn't work, then it may be something particular to my setup.

**Comment by jjurkus** [comment]
I have the same issue when I try to update chromium web store via itself.
I removed all my automatic installations, and installed 1.0.2. After restarting chromium it sees there is is an update available. Selecting "update all extensions" gives the CRX_REQUIRED_PROOF_MISSING error.
